### ISSUE TITLE
lodash dependency doesn't seem to be packaged properly

### ISSUE THREAD

**Issue by crodriguez1a** [crodriguez1a/ember-sort-filter-table#3]

**Comment by crodriguez1a** [comment]
When installing the Add-on in a project, the build fails on the LoDash dependency. Manual installation of "ember-lodash" in the project seems to correct the issue.

**Comment by alexdiliberto** [comment]
Nested addons still have issues in general which is why it works if the project just needs to use  `ember-lodash` as a dependency, but it doesn't work if my projects **addon** is using `ember-lodash` as a dependency.

In the nested case where you `ember install ember-sort-filter-table` from a fresh ember cli app...It fails on this line https://github.com/mike-north/ember-lodash/blob/master/index.js#L11 because `this.project.addonPackages['ember-lodash']` isn't a thing in the nested case (normally it is). I pinged a few people on slack and it's a known issue.

Basically the problem stems from the issue of depuping, you can't guarantee that the an addon won't contain a conflicting version in your consuming app. So if you've got an addon that uses `ember-lodash`, and you use another addon that happens to use `ember-lodash`, ember-cli doesn't have enough to go off of on which one wins.

I'll submit a PR

**Comment by alexdiliberto** [comment]
@crodriguez1a Just a note here that this should be fixed now upstream via this commit https://github.com/ef4/ember-lodash/commit/07abd88917a377c54075bff3fcd623a4b0c043f5

you can try to remove my fix and see if it still works when doing an addon install from a normal ember app

The fix is in ember-lodash@0.0.6 btw
